src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './Slider.scss';
+
+const Slider: React.FC<React.InputHTMLAttributes<HTMLInputElement>> = (props) => {
+	return (
+		<div className={`flex h-8 w-full items-center justify-center rounded`}>
+			<input
+				{...props}
+				disabled={props.disabled}
```

Comment:
@manishdoley23 no need of this, as the props are already provided in the line above